### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by entorb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +85,7 @@ top directory use the command:
 
 ## Contributing
 
-Contributions are most welcome. These fall into three main categories:
```
Comment (rrthomas) [inline README.md:87]: Still only three categories (translation is one category).

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ https://github.com/rjl20/hpmor
 Maintainer: Reuben Thomas <rrt@sc3d.org>
 
 A LaTeX version of [the popular didactic fan-fiction](http://www.hpmor.com)
-by Eliezer Yudkowsky, which can make a PDF e-book (one file) or printable
+by Eliezer Yudkowsky, which can make e-books in PDF, ePub and Mobi formats, and printable PDF
 books (either one or six volumes; the latter option is more practical to
```
Comment (rrthomas) [inline README.md:8]: I suggest "…and six PDF volumes that can be printed and bound".

File: README.md (hunk 1 above)
Comment (entorb) [reply README.md:87]: Sorry, I do not get this ;-)

Comment (rrthomas) [reply README.md:87]: Sorry, you're quite right, ignore my comment. Actually, I think this is an anti-pattern in documentation, I would suggest "These fall into the following categories:".

Comment (entorb) [reply README.md:87]: oki, changed

**Comment by rrthomas** [comment]
One more thing, please can you rewrap the lines to a sensible length? It's nice if markdown displays sensibly in the terminal as well as when rendered as HTML.

**Comment by entorb** [comment]
ok, fixed the line length as well. Shall we merge it?

File: README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ https://github.com/rjl20/hpmor
 Maintainer: Reuben Thomas <rrt@sc3d.org>
 
 A LaTeX version of [the popular didactic fan-fiction](http://www.hpmor.com)
-by Eliezer Yudkowsky, which can make a PDF e-book (one file) or printable
-books (either one or six volumes; the latter option is more practical to
-bind). There are also dust jackets for the printable volumes.
+by Eliezer Yudkowsky, which can make e-books in PDF, ePub and Mobi formats,
+and six PDF volumes that can be printed and bound.
+There are also dust jackets for the printable volumes.
```
Comment (rrthomas) [inline README.md:9]: Sorry to be a pedant: this line needs wrapping on to the previous.

**Comment by rrthomas** [comment]
One small bit of rewrapping next (also, thanks for fixing a long line that looks like one of mine!)

File: README.md (hunk 3 above)
Comment (entorb) [reply README.md:9]: Sorry, I don't get it. May I ask you to do the remaining modification yourself? I guess that is faster than ping-pong ;-)

**Comment by rrthomas** [comment]
I don't seem to have permission to push to your repo, as per https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by entorb** [comment]
strange I here it looks like that you should have permissions:

image

**Comment by rrthomas** [comment]
I wonder if this is something to do with the repo rename. Maybe let's try again when we have new PRs created since the rename. In the mean time, could you apply the following, please:

```
From aefbfb7074a7037589c67ee027589cb35152da9a Mon Sep 17 00:00:00 2001
From: Reuben Thomas <rrt@sc3d.org>
Date: Mon, 9 May 2022 09:49:00 +0100
Subject: [PATCH] Fix one more line-wrap

---
 README.md | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/README.md b/README.md
index f9447b5..79f9ea7 100644
--- a/README.md
+++ b/README.md
@@ -5,8 +5,7 @@ Maintainer: Reuben Thomas <rrt@sc3d.org>
 
 A LaTeX version of [the popular didactic fan-fiction](http://www.hpmor.com)
 by Eliezer Yudkowsky, which can make e-books in PDF, ePub and Mobi formats,
-and six PDF volumes that can be printed and bound.
-There are also dust jackets for the printable volumes.
+and six PDF volumes that can be printed and bound. There are also dust jackets for the printable volumes.
 
 TeXLive 2015 or later and git are required to build the book. (Note: the
 book must be built from a git checkout.)
-- 
2.25.1
```

**Comment by entorb** [comment]
done

**Comment by entorb** [comment]
shall we merge it?

File: README.md [hunk 4]
```diff
@@ -4,9 +4,8 @@ https://github.com/rjl20/hpmor
 Maintainer: Reuben Thomas <rrt@sc3d.org>
 
 A LaTeX version of [the popular didactic fan-fiction](http://www.hpmor.com)
-by Eliezer Yudkowsky, which can make a PDF e-book (one file) or printable
-books (either one or six volumes; the latter option is more practical to
-bind). There are also dust jackets for the printable volumes.
+by Eliezer Yudkowsky, which can make e-books in PDF, ePub and Mobi formats,
+and six PDF volumes that can be printed and bound. There are also dust jackets for the printable volumes.
```
Comment (rrthomas) [inline README.md:8]: This line still needs re-wrapping! Sorry, I still can't push to this branch for some reason.

Comment (rrthomas) [reply README.md:8]: (I think you applied my patch immediately, so didn't see that I updated it.)

Comment (entorb) [reply README.md:8]: yes, sorry, I was too lazy to google how to handle our fix, so I just copied the text.

Comment (rrthomas) [reply README.md:8]: No worries. This line still needs rewrapping!

File: README.md [hunk 5]
```diff
@@ -197,7 +198,8 @@ It is recommended to use `polyglossia` (not `babel`).
 Note: there are other translations of HPMOR; here are listed only
 translations of this edition.
 
-* [French](https://github.com/yeKcim/hpmor) (in progress)
+* [French](https://github.com/yeKcim/hpmor) (complete)
+* [German](https://github.com/entorb/hpmor-de) (volume 1 (ch1-21) finished)
```
Comment (rrthomas) [inline README.md:202]: Why not spell out "chapters 1–21"? Also wrap line 202.

File: README.md (hunk 4 above)
Comment (entorb) [reply README.md]: now than line should be fine

File: README.md (hunk 5 above)
Comment (entorb) [reply README.md]: done

**Pull request merged**